projects/igniteui-angular/src/lib/grids/grid-toolbar.component.ts

```diff
@@ -28,7 +28,6 @@ import { IgxColumnPinningComponent } from './column-pinning.component';
 import { OverlaySettings, PositionSettings, HorizontalAlignment, VerticalAlignment } from '../services/overlay/utilities';
 import { ConnectedPositioningStrategy } from '../services/overlay/position';
 
-/** @hidden */
```

Comment:
Could we put some information about the component here?

Reply:
Addressed.